ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ntp.yml

```diff
@@ -24,4 +24,5 @@ ntp:
     - id: 2
       hash_algorithm: "sha1"
       key: "15060E1F10"
+      key_type: 8a
```

Comment:
please correct the keys to be correct for the type.

Reply:
fixed invalid key